File: tutorials/code/simp/cpp/eval-pst/eval-pst.cpp
#include <linkpred.hpp>
#include <iostream>
using namespace LinkPred::Simp;
int main() {
	// Create an evaluator object
	Evaluator eval;
	// Add predictors to be evaluated
	eval.addADA();
	// Use the method addPST to create a predictor that loads scores from pst.txt
	eval.addPST("PST", "pst.txt");
	eval.addRAL();
	// Add performance measures
	eval.addPR();
	eval.addTPR();
	// Run experiment on the specified network
	eval.run("Zakarays_Train.edges", "Zakarays_Test.edges");
	return 0;
}

File: tutorials/code/simp/python/eval-pst/eval-pst.py
# Import the module
import LinkPredPython as lpp
# Create an evuator object
ev = lpp.Evaluator();
# Add predictors to be evuated
ev.addADA();
# Use the method addPST to create a predictor that loads scores from pst.txt
ev.addPST("PST", "pst.txt");
ev.addRAL();
# Add performance measures
ev.addPR();
ev.addTPR();
# Run experiment on the specified network
ev.run("Zakarays_Train.edges", "Zakarays_Test.edges");
